x-pack/plugins/cloud_security_posture/public/pages/configurations/findings_flyout/findings_flyout.tsx

```diff
@@ -164,6 +188,17 @@ export const FindingsRuleFlyout = ({
             </EuiTitle>
           </EuiFlexItem>
         </EuiFlexGroup>
+        <div
+          css={css`
+            line-height: 20px;
+            margin-top: ${euiThemeVars.euiSizeM};
+          `}
+        >
+          <CspInlineDescriptionList
+            testId={FINDINGS_VULNERABILITY_FLYOUT_DESCRIPTION_LIST}
```

Comment:
this test_id is imported from vulnerabilities, I think we should use a test id different for misconfigurations